### DISCUSSION TITLE
Colorize plugin: "Neither 'pygments' nor 'chroma' is installed!"

### DISCUSSION THREAD

**Discussion by steinbil** [ohmyzsh/ohmyzsh#12187] [Help]
I have installed oh my zsh and the colorize plugin via Antigen, but when I try to use the ccat command I get the output:
`Neither 'pygments' nor 'chroma' is installed!`

I don't see either of these plugins on the list/directory of oh my zsh plugins. Do these plugins have dependencies that are not part of oh my zsh? I assumed the plugins included with oh my zsh would be self-contained, but maybe that is a misconception from my side? Or is there something else that might have gone wrong with my setup?

**Comment by rwmitchell** [comment]
It's a misconception on your part.

Generally, plugins augment usage of programs already on your system.  In some cases, you may need to install them.

For example, plugins for docker do not provide docker, they just make it easier to use.

**Reply by steinbil** [reply]
Understood. The reason I was confused is not the idea of plugins, but the idea of ohmyzsh as a concept. I assumed that ohmyzsh as a framework didn't have external dependencies other than the actual zsh shell. Or stated more explicitly: if Colorize is an included plugin in this framework, why are not the dependencies also included? Or at least a documented list of external dependencies for the framework (if this exists, I've missed it)?

**Reply by nyaahilism** [reply]
[https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/colorize](url)

It actually lists them on the plugin page for Colorize. Hope this helps (꒪ˊ꒳ˋ꒪)ꕤ*.ﾟ
